packages/commonwealth/server/routes/tokenBalance.ts

```diff
@@ -63,7 +63,7 @@ const tokenBalance = async (
     } else if (e.message === FetchTokenBalanceErrors.UnsupportedContractType) {
       throw new AppError(FetchTokenBalanceErrors.UnsupportedContractType);
     } else {
-      throw new ServerError(Errors.QueryFailed);
+      throw new ServerError(Errors.QueryFailed, e);
```

Comment:
What does it mean to add the second error arg here?

Reply:
Isn't that so that the `underlyingError` prop on `ServerError` isn't undefined?

Reply:
Exactly:
```
export class ServerError extends Error {
  status: number;
  // Optionally include the original error that was thrown
  error: Error;
  constructor(message: string, error?: Error) {
    super(message);
    this.status = 500;
    this.name = 'ServerError';
    this.error = error;
  }
}
```